assignments/lambda-classes.js

```diff
@@ -1 +1,105 @@
 // CODE here for your Lambda Classes
+class Person {
```

Comment:
This is causing me so much nostalgia from my teenage years.
Great code and great content!